edm4hep.yaml

```
@@ -46,6 +46,38 @@ components:
         bool operator==(const Vector2i& v) const { return (a==v.a&&b==v.b) ; }\n
         int operator[](unsigned i) const { return *( &a + i ) ; }\n
         "
+      # Parametrized description of a particle track
+    edm4hep::TrackState:
+      location       : int # for use with At{Other|IP|FirstHit|LastHit|Calorimeter|Vertex|LastLocation}
```

```suggestion
      location       : int # for use with At{Other|IP|FirstHit|LastHit|Calorimeter|Vertex}|LastLocation
```
LastLocation does not start with `At`